pom.xml

```diff
@@ -106,15 +106,15 @@
     <beam.version>2.0.0</beam.version>
 
     <dataflow.release_name>Google Cloud Dataflow SDK for Java</dataflow.release_name>
-    <dataflow.container_version>${project.version}-20170317</dataflow.container_version>
+    <dataflow.container_version>${project.version}-20170517</dataflow.container_version>
     <dataflow.environment_major_version>6</dataflow.environment_major_version>
 
     <!-- Archetype version propagated automatically to underlying archetype modules.
          Example values are: ${project.version}, 1.2.3-SNAPSHOT, [1.0.0,2),
                              [${project.version}, 2), etc.
          Starter archetype reference copy isn't affected by this value; a manual
```

Comment:
Yes, this happened in https://github.com/GoogleCloudPlatform/DataflowJavaSDK/pull/569 and is at odds with both the comment and the value in the root pom and archetype resources pom.

Maybe just delete the variable and manage everything on `project.version`?